### PR TITLE
Tec 2853

### PR DESCRIPTION
**Pull request by joae**
Added cart to the library. 
version 1.0.13 published

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ class Example extends Component {
 ## Deploy
 Update version number in package.json and then run:
 ```sh
-npm deploy
+npm run publish
+npm run deploy
```
Comment (patriciomase) [inline README.md:47]: sweet!

File: example/src/messages.js [hunk 2]
```diff
@@ -16,8 +15,30 @@ const messages = {
     "header.search": "Search",
     "header.search.placeholder": "Find what you're looking for",
     "header.search.hint": "Search any products and/or materials",
-    "header.search.results": "Results"
+    "header.search.results": "Results",
+
+    "cart.header.advice": "You're ${label} away from free shipping",
```
Comment (patriciomase) [inline example/src/messages.js:20]: Is this available if I just import the Cart component?

Comment (patriciomase) [reply example/src/messages.js:20]: I think if we have react-intl in the other repository too things are going to mix up

File: src/components/cart/Cart/components/CartSummary/components/Adjustments/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage } from 'react-intl'
+import { List, Row } from '../../styled'
+import { toCurrencyWithCode } from '../../../../../../../helpers/currency'
```
Comment (patriciomase) [inline src/components/cart/Cart/components/CartSummary/components/Adjustments/index.js:5]: :P we really need to make absolute imports to work, I can take a look if needed

File: example/src/messages.js (hunk 2 above)
Comment (joae) [reply example/src/messages.js:20]: NO it isn't. It belongs to the example project. We still need to add texts in the project that use it

File: src/components/Header/components/ProductCard/styled.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
-import { bp } from 'packages/styled'
-import colors from 'app/styles/colors'
-import { fontWeight } from 'app/styles/settings'
+import { bp } from 'app/packages/styled'
+import colors from '../../../../styles/colors'
```
[the remark this answers is not in the document]
Comment (joae) [reply src/components/Header/components/ProductCard/styled.js:3]: Yes, sure. I tried with not so much effort. I'll revisit it

**Comment by joae** [comment]
@patriciomase branch updated using absolute path :)

**Pull request merged**